_trophy/2.2017.md

```diff
@@ -4,27 +4,27 @@ achievements_students:
   - |
     - Team PlutoAR wins UnitedByHCL Hackathon in London (team comprised of Ankit Panda, Shourya Pratap Singh and ShushobhitPanigrahi)
   - |
-    - Shibasis Patel selected as a Google Summer of Code Student under Open Astronomy
+    - Shibasis Patel is selected as a Google Summer of Code Student under Open Astronomy
   - |
-    - Dibyajyoti Panda selected as a Google Summer of Code Student under Drupal
+    - Dibyajyoti Panda is selected as a Google Summer of Code Student under Drupal
   - |
-    - Dibyajyoti Panda selected as a Google CodeIn Mentor under Drupal
+    - Dibyajyoti Panda is selected as a Google CodeIn Mentor under Drupal
   - |
-    - Shubham Rath, Aman Kumar Gupta & Shivank Tripathi selected for ACM ICPC Kolkata Regionals
+    - Shubham Rath, Aman Kumar Gupta & Shivank Tripathi are selected for ACM ICPC Kolkata Regionals
   - |
-    - Prateek Pattnaik, Nawed Imroze and Bibhu Pala selected for ACM ICPC Amritapuri Regionals
+    - Prateek Pattnaik, Nawed Imroze and Bibhu Pala are selected for ACM ICPC Amritapuri Regionals
   - |
-    - Team PiedPiper (someone please fill in the names of the members) won the first prize in Smart India Hackathon.
+    - Team PiedPiper (someone please fill in the names of the members) win the first prize in Smart India Hackathon.
   - |
-    - Team "The Web Samaritans" won the second runners-up in Smart India Hackathon(members:Abhijeet Panda, Anuva Acharya, Sheikh  - Sohail, Sami Ur Rehman, Rohan Panda, Shubhankar Mohapatra)
+    - Team "The Web Samaritans" (members: Abhijeet Panda, Anuva Acharya, Sheikh - Sohail, Sami Ur Rehman, Rohan Panda, Shubhankar Mohapatra) win the second runners-up in Smart India Hackathon
   - |
-    - Another team were the finalist in GE Healthhack can someone fill in their names please (cc @sr6033 @shibasisp )
+    - Another team become finalist in GE Healthhack can someone fill in their names please (cc @sr6033 @shibasisp )
```

Comment:
We need this removed `(cc @sr6033 @shibasisp )` I guess some careless contributor just copy pasted the things I had written on an issue thread just remove any *cc's* or @ mentions you find @itsmepiyush2

Reply:
Okay